.circleci/config.yml

```diff
@@ -120,9 +120,6 @@ workflows:
             tags:
               only: /^v.*/
           requires:
-            - build
-            - harness
-            - harness-macos
```

Comment:
are we sure just requiring `publish` will also wait for these steps? I mean, I know these steps ARE required by `publish` step, but is it safe to assume so?

Reply:
Yes, according to the CircleCI documentation.